tools/manifest/sourcefile.py

```diff
@@ -49,6 +49,86 @@ def read_script_metadata(f, regexp):
         yield (m.groups()[0], m.groups()[1])
 
 
+_any_variants = {
+    b"window": {"default": True, "suffix": ".any.html"},
```

Comment:
Thought: maybe we should instead have `b"default": {"longhand": {b"window", b"dedicatedworker"}}`, then you could use
```js
// META: global=!default,blah
```